projects/rero/ng-core/src/lib/record/record-ui.service.ts

```diff
@@ -71,13 +74,18 @@ export class RecordUiService {
           return of(false);
         }
 
+        this._spinner.show();
+
         return this._recordService.delete(type, pid).pipe(
           map(() => {
-            this._toastService.success(this._translateService.instant('Record deleted.'));
             return true;
           }),
           // delay before doing anything else, otherwise records may be not refreshed.
-          delay(1000)
+          delay(1000),
+          tap(() => {
+            this._spinner.hide();
```

Comment:
Is it really the role of a service to hide the sipnner? This seems to be done in a component.

Reply:
I didn't do this change because the spinner have to be displayed after confirmation modal and the display of the modal is handled in `record-ui.service.ts`. That's why we suffixed it with `ui`. 
I agree that could be improved, but we can do that in another PR.